lit_gpt/config.py

```diff
@@ -12,8 +12,8 @@
 
 @dataclass
 class Config:
-    org: str = "Lightning-AI"
     name: str = "lit-GPT"
+    hf_config: Optional[dict] = None
```

Comment:
I don't think this should be optional. At least for now while all our checkpoints are backed by a HF config. Making it required avoids the issue of none checks everywhere

Reply:
Hmm, I disagree because that was the whole point of the PR, to make it optional. 

> I want to make my own GPT model using the awesome lit-gpt template. So I make my own config but oh... I need to provide a hf configuration? But I haven't trained my model yet, should I put in fake data?